roles/openshift_master/handlers/main.yml

```diff
@@ -6,6 +6,7 @@
   when:
   - not (master_api_service_status_changed | default(false) | bool)
   - openshift.master.cluster_method == 'native'
+  - master_api_enabled.rc == 0 or openshift_version | version_compare('3.7','>=')
```

Comment:
I wonder if there may be cases where this code is called and this variable may be undefined? Perhaps
` - master_api_enabled is defined and (master_api_enabled.rc == 0 or openshift_version | version_compare('3.7','>='))`

Reply:
Thank you. Yes, it is safer.

I wondered if I should write these as:

~~~
  - master_api_enabled is defined
  - master_api_enabled.rc == 0 or openshift_version | version_compare('3.7','>=')
~~~

But maybe `- master_api_enabled is defined and (master_api_enabled.rc == 0 or openshift_version | version_compare('3.7','>='))` is easier to understand why `master_api_enabled is defined` is checked.